### PR TITLE
Added backend for the Codebadge project

### PR DESCRIPTION
**Pull request by ayushnagar123**
### Problem
Github Issue Number: #53 #51 

### PR includes:
- [X] login
- [X] signup
- [X] fetching user details
- [X] fetching organization details
- [X] fetching organization members
- [X] Codebadge bot fetching all events through webhook

Type of Change
- [ ] Bug fix
- [X] New Feature
- [ ] Development of UI/UX prototypes
- [ ] Small refactor
- [X] Change in Documentation

Checklist
- [X] My code follows the same style as the codebase
- [X] My Code change requires a change in documentation
- [X] I have updated the Readme accordingly
- [X] I made PR against development branch
- [X] I have run the test cases locally and it's passing.
- [ ] I have squashed my commits

### CODE REVIEW

**Comment by jaskiratsingh2000** [comment]
Good

**Comment by devesh-verma** [comment]
1. @ayushnagar123 Going ahead I would like to see PRs more often. It's hard to review 92file changes in 1PR. Also, it's not preferred keeping in consideration the commit history and progress of the project.
2. Make 1PR per functionality.
3. Also, work on your commit messages. 
4. Squash your commits.

TL;DR
I am okay reviewing more PRs rather than reviewing 1PR with so much of work.

**Comment by ayushnagar123** [comment]
Surely i will take care of it next time thanks for the inputs.

**Comment by ayushnagar123** [comment]
I have updated and created a new pull request #55
